src/actions/starMessage.js

```diff
@@ -0,0 +1,26 @@
+import { getRoomsChannel } from '../reducers/rooms'
+
+export const starMessage = (messageId) => ({
+  type: 'STAR_MESSAGE',
+  messageId: messageId
+})
+
+export const unstarMessage = (messageId) => ({
+  type: 'UNSTAR_MESSAGE',
+  messageId: messageId
+})
+
+export const replaceStarMessages = (data) => ({
+  type: "REPLACE_STAR_MESSAGES",
+  starredMessages: data
+})
+
+export const submitStar = (messageId) => (dispatch, getState) => {
+  const rooms = getRoomsChannel(getState())
+  return rooms.push('starred_message:create', messageId)
```

Comment:
Two formatting rules:

- Linebreak after variable (`const`, `let`, and `var`) definitions
- Linebreak before `return` statements

So in this case, need an empty line above `return` :)